### PR TITLE
Unify ims init and fix error handling

### PR DESCRIPTION
**Pull request by karlpauls**
We now load ims in several places and don't handle the case where we get an error (due to the domain size limit). This PR fixes this.

### CODE REVIEW

File: blocks/edit/da-editor/da-editor.js [hunk 1]
```diff
@@ -18,9 +15,11 @@ export default class DaEditor extends LitElement {
     super.connectedCallback();
     this.shadowRoot.adoptedStyleSheets = [sheet];
     this.shadowRoot.createRange = () => document.createRange();
-    loadIms().then(() => {
-      this._imsLoaded = true;
-    });
+    if (!this._imsLoaded) {
```
Comment (auniverseaway) [inline blocks/edit/da-editor/da-editor.js:18]: Is this necessary? Connected callback should only be called when the component is attached to the DOM... which would mean this is never true.

https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements#:~:text=called%20each%20time%20the%20element%20is%20added%20to%20the%20document.

**Review by auniverseaway — APPROVED** [review]
I think our IMS handling could still use improvement / be more consistent, and I'm not sure this solves the issues I _think_ we have, but it's a step in the right direction.

Merge away when you're ready.

File: blocks/edit/da-editor/da-editor.js (hunk 1 above)
Comment (karlpauls) [reply blocks/edit/da-editor/da-editor.js]: Good point, removed it.

**Comment by karlpauls** [comment]
Yeah, it at a minimum does not solve the issue where the domain is too long - but at least it does not make use fail in that case anymore.

**Pull request merged**